CONTRIBUTING.md

```diff
@@ -40,27 +40,42 @@ of OCaml language item that needs to be displayed in docs.
 
 To make edits to the HTML generation, run the following commands:
 
-1. Set up for development:
+1. Install requirements:
+
+    - A recent version of [tidy](http://www.html-tidy.org/) (used for
+      HTML validity testing) is required:
+
+        ```
+        # On MacOS (should be version 5.6.0 by the date of this writing)
+        brew install tidy-html5
+        
+        # Debian / Ubuntu
+        sudo apt-get install tidy
+        ```
+
+    - A recent version of [jq](https://github.com/stedolan/jq) is required.
+
+        ```
+        # On MacOS
+        brew install jq
+        
+        # Debian / Ubuntu
+        sudo apt-get install jq
+        ```
+2. Set up for development:
 
     ```
-    git clone https://github.com/ocaml/odoc.git
+    git clone --recurse-submodules https://github.com/ocaml/odoc.git
     cd odoc
     opam pin add --no-action odoc .
-    opam install --deps-only odoc
+    opam install --with-test --deps-only odoc
+    opam install --deps-only mdx
```

Comment:
The `mdx` submodule is meant to fix a dependency cycle between odoc and mdx, which was problematic in OCaml-CI. It's temporary.
I think it's fine to do `opam install mdx` and to not init the submodule.

Reply:
Ok.  I can update the PR to just install mdx.  What do you prefer?

Reply:
Let's leave it as-is. Once we've got rid of the submodule we can remove this instruction.